app/models/match_decisions/homeless_set_aside/hsa_acknowledges_receipt.rb

```diff
@@ -0,0 +1,120 @@
+module MatchDecisions::HomelessSetAside
+  class HsaAcknowledgesReceipt < ::MatchDecisions::Base
+
+    include MatchDecisions::AcceptsDeclineReason
+
+    validate :cant_accept_if_match_closed
+    validate :cant_accept_if_related_active_match
+    validate :ensure_required_contacts_present_on_accept
+
+    def label_for_status status
+      case status.to_sym
+      when :pending then "New Match Awaiting Acknowledgment by #{_('HSA')}"
+      when :acknowledged then "Match acknowledged by #{_('HSA')}.  In review"
+      when :canceled then canceled_status_label
+      end
+    end
+
+    def started?
+      status&.to_sym == :acknowledged
+    end
+
+    def step_name
+      "New Match for #{_('HSA')}"
+    end
+
+    def actor_type
+      _('HSA')
+    end
+
+    def contact_actor_type
+      nil
+    end
+
+    def statuses
+      {
+        pending: 'Pending',
+        acknowledged: 'Acknowledged',
+        canceled: 'Canceled',
+      }
+    end
+
+    def editable?
+      super && saved_status !~ /acknowledged/
+    end
+
+    def initialize_decision! send_notifications: true
+      super(send_notifications: send_notifications)
+      update status: :pending
+      send_notifications_for_step if send_notifications
+    end
+
+    def accessible_by? contact
+      contact.user_can_act_on_behalf_of_match_contacts? ||
+      contact.in?(match.housing_subsidy_admin_contacts)
+    end
+
+    def notifications_for_this_step
+      @notifications_for_this_step ||= [].tap do |m|
+        m << Notifications::HomelessSetAside::MatchInitiationForManualActivation
```

Comment:
Pretty sure `MatchInitiationForManualActivation` isn't defined anywhere.  I think this should remain `MatchInitiationForHsa` as the manual activation notification is one step before this.

Reply:
Looks like maybe this was supposed to be attached to `MatchInitiationForManualNotification`